lib/base/RequestClient.js

```diff
@@ -94,6 +95,9 @@ RequestClient.prototype.request = function(opts) {
   this.lastRequest = new Request(optionsRequest);
 
   axios(options).then((response) => {
+    if (opts.debug) {
+      debugLogRequest(response)
```

Comment:
Should also look at logging the response.

Reply:
In addition to the parsed response in `debugLogRequest` also logging the whole response for users like: `console.log(response)`?

Reply:
Wait, I was confused. This is called `debugLogRequest` but it's logging the response. That should be fixed.

I meant why not log the request _and_ the response?